src/plugins/liveobjects/statemessage.ts

```
@@ -0,0 +1,309 @@
+  }
+
+  static fromValues(values: StateMessage | Record<string, unknown>, platform: typeof Platform): StateMessage {
+    return Object.assign(new StateMessage(platform), values);
+  }
+
+  static fromValuesArray(values: unknown[], platform: typeof Platform): StateMessage[] {
+    const count = values.length;
+    const result = new Array(count);
+
+    for (let i = 0; i < count; i++) {
+      result[i] = this.fromValues(values[i] as Record<string, unknown>, platform);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)